queries.sql

```diff
@@ -0,0 +1,9 @@
+
+SELECT * from animal WHERE name LIKE '%mon';
+SELECT name FROM animal WHERE date_of_birth BETWEEN '1-1-2016' AND '1-1-2019';

```

Comment:
- [ ] As the previous reviewer requested, please make sure this query ends on `31-12-2019` in order to include that month in the year that the query is asking for.

Hint:

```sql
SELECT name FROM animal WHERE date_of_birth BETWEEN '1-1-2016' AND '31-12-2019';
```